src/points/PointsHome.tsx

```diff
@@ -48,14 +52,31 @@ export default function PointsHome({ route, navigation }: Props) {
       onPress()
     }
   }
+  const onPressActivity = () => {
+    ValoraAnalytics.track(PointsEvents.points_screen_activity_press)
+    dispatch(getHistoryStarted({ fromPage: false }))
```

Comment:
what do you think about dispatching this action on screen mount, to reduce the chances of users needing to wait to see the activity feed? of course we could argue that the activity feed might not be launched but it's a background task and doesn't require much bandwidth to make the network request